Rename inplace variant of freeze_module

Gamrix opened the pull request:
The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.

Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71437
* #71436

Differential Revision: [D33824856](https://our.internmc.facebook.com/intern/diff/D33824856)

Gamrix committed:
Proposal to further differentiate the API

[ghstack-poisoned]

Gamrix committed:
Update on "Proposal to further differentiate the API"




[ghstack-poisoned]

Gamrix committed:
Update on "Rename inplace variant of freeze_module"

The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.




[ghstack-poisoned]

Gamrix committed:
Update on "Rename inplace variant of freeze_module"

The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.




[ghstack-poisoned]

Gamrix committed:
Update on "Rename inplace variant of freeze_module"

The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.




[ghstack-poisoned]

Gamrix committed:
Update on "Rename inplace variant of freeze_module"

The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.




[ghstack-poisoned]

eellison reviewed (approved):
LGTM, but you'll need to make internal changes as well

Gamrix commented:
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33824856).

Gamrix committed:
Update on "Rename inplace variant of freeze_module"

The two versions of freeze_module can be easily mixed up. This is to make the distinction more clear.


Differential Revision: [D33824856](https://our.internmc.facebook.com/intern/diff/D33824856)

[ghstack-poisoned]
